samples/cellular/nrf_cloud_multi_service/src/shadow_config.c

```diff
@@ -196,6 +196,8 @@ int shadow_config_accepted_process(struct nrf_cloud_obj *const accepted_obj)
 	if (err == 0) {
 		accepted_rcvd = true;
 	}
-
+	if (err) {
+		LOG_ERR("Error: %d", err);
```

Comment:
`process_cfg` prints error logs... you could add the return value into those instead

Reply:
Not necessary. Will remove.